app/views/admin/worldwide_organisations_translations/index.html.erb

```diff
@@ -1,55 +1,82 @@
-<% page_title @worldwide_organisation.name + " translations" %>
+<% content_for :page_title, "#{@worldwide_organisation.name} translations" %>
+<% content_for :title, @worldwide_organisation.name %>
+<% content_for :context, "Organisation" %>
+<% content_for :title_margin_bottom, 4 %>
 
-<%= content_tag_for(:div, @worldwide_organisation) do %>
-  <div class="organisation-header">
-    <h1><%= @worldwide_organisation.name %></h1>
-    <%= view_on_website_link_for @worldwide_organisation %>
+<div class="govuk-grid-row">
```

Comment:
does this p tag need the two thirds grid column around it? As it is left aligned and short?